### PR TITLE
improve use case

### PR DESCRIPTION
**Pull request by TonPC64**

### CODE REVIEW

**Comment by sepiropht** [comment]
Yes ! Thank for that, this is my first review on github !
Could you please squash your commit in one ? It will help my review.
Thanks again for the help.

**Comment by TonPC64** [comment]
add feature load one video

**Comment by TonPC64** [comment]
sorry for my language is bad.

**Comment by sepiropht** [comment]
Ok well, what i ask is that  :[squash commit](https://makandracards.com/makandra/527-squash-several-git-commits-into-a-single-commit)

**Comment by TonPC64** [comment]
squash done.

File: README.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
 # Frontend-Master-Downloader
 Scrape frontendmaster with puppeteer (Not fully working for now, just try it for fun)
 
+### Usage
+
+**Requirements**
+
+- Node Ver.LTS
```
Comment (sepiropht) [inline README.md]: Requirement should be  "node v7.6.0 or greater". because we use async/await

File: index.js [hunk 2]
```diff
@@ -89,43 +104,81 @@ mkdirp(directory, function(err) {
...
-
-  console.log("Will start downloading videos");
+    const fileName = `${index+1}-` + link.split("/").filter(str => str.length).pop() + ".mp4";
+    try {
+      downloadVideo({ fileName, videoLink })
+    } catch (err) {
+      console.log(err)
+    }
+  } else {
+    for (const templink of newLink.reverse()) {
```
Comment (sepiropht) [inline index.js]: I don't understand why you reverse the list

Comment (TonPC64) [reply index.js]: sorry I mistake.

**Pull request merged**